pkg/controller/controller_broker.go

```diff
@@ -299,44 +299,44 @@ func (c *controller) reconcileClusterServiceBroker(broker *v1beta1.ClusterServic
 		existingServiceClassMap := convertServiceClassListToMap(existingServiceClasses)
 		existingServicePlanMap := convertServicePlanListToMap(existingServicePlans)
 
-		// reconcile the plans that were part of the broker's catalog payload
```

Comment:
The diff is misleading. What I have actually done is just moved the "serviceclasses" block before the "plans" block. No other code change.